libretro-common/include/retro_endianness.h

```diff
@@ -95,6 +95,8 @@
 #  endif
 #elif defined (__i386__) || defined(__x86_64__)
 #  define LSB_FIRST 1
+#elif defined (__CELLOS_LV2__)
```

Comment:
I think this can be handled in the makefile?

Reply:
Done